### DISCUSSION TITLE
Coherent Network Growth

### DISCUSSION THREAD

**Discussion by wangzhao0217** [nptscot/npt#504] [Ideas]
Hi @Robinlovelace,

How does this growing look?
![image](https://github.com/nptscot/npt/assets/74598734/3dff0dcc-8475-48ba-94d5-ae69235d1035)
![image](https://github.com/nptscot/npt/assets/74598734/99c29348-cdd3-4f2d-88ac-1d746a5c18e8)
![image](https://github.com/nptscot/npt/assets/74598734/d6b8ed07-3225-4745-8b64-172bde20b715)

**Comment by Robinlovelace** [comment]
Looks good Can it grow in more smaller steps @wangzhao0217? BTW good use of Discussions!

**Reply by wangzhao0217** [reply]
Yes, certainly, now the step is like 5000, 3000 and 1500
